lidder.go

```diff
@@ -246,9 +247,9 @@ func singleFileMode(configFilename, filename string) bool {
 		if len(shouldNotBeThere) != 0 || len(shouldBeThere) != 0 {
 			success = false
 			if len(shouldNotBeThere) != 0 {
-				fmt.Printf("Lidded pattern '%s' found\n", rule.Pattern)
+				fmt.Printf("Lidded rule '%s' (pattern '%s') found\n", rule.Name, rule.Pattern)
```

Comment:
Suggest

```
func (rule *rule) String() {
  return fmt.Sprintf("Lidded rule '%s' (pattern '%s')", rule.Name, rule.Pattern)
}
```
and then here `fmt.Printf("%s found\n", rule)`.

Also, since you're changing the syntax to use `name: ...`, you should handle a nice message if it is an anonymous (nameless) rule. Or, require to have a non-empty name.

Reply:
👍